app/src/main/java/org/mozilla/focus/browser/BrowserFragment.kt

```diff
@@ -165,25 +170,51 @@ class BrowserFragment : IWebViewLifecycleFragment() {
             onNavigationEvent = this@BrowserFragment.onNavigationEvent
             navigationStateProvider = NavigationStateProvider()
             visibility = overlayVisibleCached ?: View.GONE
+            openHomeTileContextMenu = {
+                activity.openContextMenu(browserOverlay.tileContainer)
+            }
             onPreSetVisibilityListener = { isVisible ->
                 // The overlay can clear the DOM and a previous focused element cache (e.g. reload)
                 // so we need to do our own caching: see FocusedDOMElementCache for details.
                 if (!isVisible) { webView?.focusedDOMElement?.cache() }
             }
+            registerForContextMenu(browserOverlay.tileContainer)
```

Comment:
nit: group this together with `openHomeTileContextMenu`:
```kotlin
...
onPerSet...

openHomeTileContext = ...
registerForContext ...
```

The reason I think this is important is because this method is starting to get long. After 4 or 5 lines (since people can only keep [~4-7 things in their head at once](https://en.wikipedia.org/wiki/The_Magical_Number_Seven,_Plus_or_Minus_Two)), it's important to start grouping.